### ISSUE TITLE
Inlining output JSON to HTML

### ISSUE THREAD

**Issue by aaronte** [ztoben/assets-webpack-plugin#74]
Is there a way to inline the output of the `assets.json` to the `index.html` and assign it to the `window` object? Like how https://github.com/soundcloud/chunk-manifest-webpack-plugin does it?
i.e. Add this to your HTML code
```js
<script>
window.webpackAssets = /* asset.json contents */
</script>
```

**Comment by ztoben** [comment]
I think this is what you're looking for:
https://github.com/kossnocorp/assets-webpack-plugin#includemanifest

**Comment by kilgarenone** [comment]
> 
> 
> I think this is what you're looking for:
> https://github.com/kossnocorp/assets-webpack-plugin#includemanifest

I tried to access `assets` object in my `index.html`, but it's not defined. How do I pass manifest.json content to html-webpack-plugin's index.html?

**Comment by ztoben** [comment]
@kilgarenone `html-webpack-plugin` doesn't have an index.html? I'm not sure what you're asking for. Could you provide some more information on your setup and what you're trying to do?

**Comment by kilgarenone** [comment]
@ztoben Sorry. I'm using this plugin in the 'multi-compiler-mode'. And I have disabled the auto-injection in `html-webpack-plugin` so I can manually inject the scripts(legacy and modern) myself. And to do that, I'm hoping that I will have be able to get my chunks' name from the `assets` object that's enabled by the [`includeManifest`](https://github.com/ztoben/assets-webpack-plugin#includemanifest) option. 

I was referred to this plugin from [this article](https://calendar.perfplanet.com/2018/doing-differential-serving-in-2019/) about 'differential serving'. But I would like to avoid the hacky injection the article suggests. I'd like to stick to 'html-webpack-plugin'. So I wonder if this plugin can lay out all my chunks(legacy and modern) as in the `htmlWebpackPlugin.files` object.

**Comment by ztoben** [comment]
@kilgarenone ah, okay. How are you loading the json file into your index file? Without doing some kind of import or reading the file I would expect that `assets` would be undefined.

**Comment by kilgarenone** [comment]
@ztoben um I just tried to access `assets` like I would for the `htmlWebpackPlugin.files` in my `index.html` assuming it'd have something there for me...

How do I 'load' or 'import' the manifest into 'index.html'? The `includeManifest` doc doesn't seem to say about this part.

**Comment by ztoben** [comment]
This option was added before I started maintaining the project, so I'm not entirely sure how it was intended to be used. All this plugin does is generate the json file, so you'd still have to import it in some way if you want to interact with it. I don't think native js allows you to import json files directly, so you'd have to parse it. Potentially with something like jQuery's [getJSON](https://api.jquery.com/jQuery.getJSON).

Sorry if that's not helpful, but that's really all the feedback I've got for you.
